packages/entando-apimanager/modules/api/APIProvider.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { config } from './apiManager';
+import { setApi } from '../state/api/actions';
+
+class APIProvider extends React.Component {
+  constructor(props) {
+    super(props);
+    this.initApiManager();
+  }
+
+  initApiManager() {
```

Comment:
why isn't this in `componentDidMount` ?

Reply:
I did try placing this in `componentDidMount` but this ends up running late after other nested components' `componentDidMount` method. So this ensures that `initApiManager` executes firsthand before anything else. Anyhow, this was from Giuseppe's work on Entando PWA and I'm pretty sure he did this way because of the particular reason that I experienced.